app/Auth/PhotoAuthorisationProvider.php

```diff
@@ -356,13 +356,13 @@ private function prepareModelQueryOrFail(FixedQueryBuilder $query, bool $addAlbu
 			$query->leftJoin('base_albums', 'base_albums.id', '=', 'photos.album_id');
 		}
 		if ($addShares) {
-			$userID = AccessControl::id();
+			$userId = Authorization::idOrNull();
```

Comment:
I assume this one should be `idOrFail`.

Reply:
Does not really matters actually. Because `Auth::check()` is guarding the condition.

Reply:
Yes, it does not matter _now_. But if it is refactored at some point in the future we should be on the safe side. Moreover, if we know that the exception is not thrown (for now), then using `idOrFail` provides us with the stricter return type.